R/share.R

```diff
@@ -42,22 +42,28 @@ setMethod("is.editor", "PermissionTuple", function (x) {
 #' of the number of emails given.
 #' @param notify logical: should users who are getting new privileges on this
 #' dataset be sent an email informing them of this fact? Default is
+#' @param message character: what message should users who are getting notified
+#' about new permissions on this dataset receive?
```

Comment:
Also, see the travis build failure for this pull request. You will need to update the man pages as well. `make doc` from the command line, or otherwise call roxygen to do that.